static/js/flow/enrollmentTypes.js

```diff
@@ -13,4 +13,5 @@ export type ProgramEnrollmentsState = {
   getStatus?: string,
   getErrorInfo?: APIErrorInfo,
   postStatus?: string,
+  coursePostStatus?: string,
```

Comment:
I agree that it's a bad place for this information, I'll go with the rename of `coursePostStatus` in the interest of moving things along since I can't think of anything better right now